docs/faq.rst

```diff
@@ -57,7 +57,7 @@ environment, and will be set to ``True`` when building on RTD::
     {% endif %}
 
 I get import errors on libraries that depend on C modules
-----------------------------------------------------------
+---------------------------------------------------------
```

Comment:
Did you remove this for some reason? Just curious :)

Reply:
@humitos the header is larger than the text p:

Reply:
Oh, that's OK. Github doesn't show it in a monospace font for some reason:

![captura de pantalla_2018-02-16_09-18-14](https://user-images.githubusercontent.com/244656/36311757-53389740-12fa-11e8-9a73-971245d71c82.png)

Reply:
I use [rstcheck](https://github.com/myint/rstcheck) for linting rst files :)

Reply:
Wow! I didn't know it and I'm a fan of @myint.

Considering that we are talking about the documentation style, I think it would be a good addition for travis checks and linting right? I mean, trigger a linting on the docs on travis checks.

Would you like to add an issue for this and take a look on what's needed and how to implement it?

Reply:
Sure, I opened a new issue #3622, I will implemented later :)